test/functional/source_file_gpx_3d.js

```diff
@@ -6,7 +6,17 @@ describe('source_file_gpx_3d', function _() {
         result = await loadExample('examples/source_file_gpx_3d.html', this.fullTitle());
     });
 
-    it('should run', async () => {
+    it('view initialized', async () => {
         assert.ok(result);
     });
+
+    it('should wait for the mesh to be added to the scene', async () => {
+        await page.waitForFunction(() => view.scene.children.length === 5);
+
+        const nbChildren = await page.evaluate(() => {
+            console.log();
+            return view.scene.children.length;
+        });
+        assert.equal(nbChildren, 5);
```

Comment:
Is all that necessary since `await page.waitForFunction(() => view.scene.children.length === 5);` is going to timeout if `view.scene.children.length != 5`.
Is it to check if `nbChildren` is not superior to 5 ? Can this value change after the `await page.waitForFunction(...)` ?

WDYT